chrome/content/zotero/standalone/standalone.js

```diff
@@ -620,6 +620,25 @@ const ZoteroStandalone = new function() {
 	};
 	
 	
+	this.promptForRestartInSafeMode = function () {
+		let ps = Services.prompt;
+		let buttonFlags = ps.BUTTON_POS_0 * ps.BUTTON_TITLE_IS_STRING
+			+ ps.BUTTON_POS_1 * ps.BUTTON_TITLE_CANCEL;
+		let index = ps.confirmEx(
+			null,
+			Zotero.getString('general.restartInSafeMode'),
+			Zotero.getString('general.restartInSafeMode.description', Zotero.appName),
```

Comment:
Probably better to create Fluent strings and use `await document.l10n.formatValues(id)` (and then `{ -app-name }` is just a Fluent term).

https://searchfox.org/mozilla-esr102/rev/c5b0b4a12acb2f13c962d748a575cad54f3ec738/dom/webidl/Localization.webidl#104-117